### DISCUSSION TITLE
Background Colour for Feature Pages

### DISCUSSION THREAD

**Discussion by acantley** [adesokan01/T24S2T1EXTG01#3] [General]
Do we have any thoughts/ideas for what background colour would be appropriate for the Mortgage Calculator? For now I have it as plain white but open to suggestions.

**Comment by adesokan01** [comment]
Thanks for reaching out on this.

Regarding the background colour, I reckon we keep it simple. 

I suggest we use : Light Gray (#F5F5F5): Light gray provides a neutral, modern look and creates a soft contrast with dark text. It’s less harsh than white, which can be easier on the eyes during extended use. It is ideal for a clean and unobtrusive background that lets other UI elements stand out.

Thanks

**Reply by acantley** [reply]
Thanks for the reply. I agree with keeping it simple and modern. I'll update the background colour to #F5F5F5.
